.travis.yml

```diff
@@ -1,104 +1,99 @@
-    - sudo ulimit -n 10000
-    - brew tap wix/brew
-    - brew install applesimutils
-    - nvm install $NODE_VERSION
-    - nvm use $NODE_VERSION
-    - nvm alias default $NODE_VERSION
-    - brew install yarn --without-node
-    - npm install -g react-native-cli
-    - npm install -g detox-cli
-    - cd e2e
-    - yarn
-    - export PRODUCT_BUNDLE_IDENTIFIER="com.swmansion.rn_gesture_handler_example"
-    - FORCE_BUNDLING=1 detox build --configuration ios.sim.debug > /dev/null
-    # lines below was done in order to kill server run by build
-    # since we require to run server with custom config
-    - killall -9 node > /dev/null &
-    - yarn start --reset-cache &
+      install:
+        - nvm --version
+        - echo $NODE_VERSION
+        - sudo launchctl limit maxfiles 2048 unlimited
+        - sudo ulimit -n 10000
+        - brew tap wix/brew
+        - brew install applesimutils
+        - nvm install $NODE_VERSION
+        - nvm use $NODE_VERSION
+        - nvm alias default $NODE_VERSION
+        - brew install yarn --ignore-dependencies
+        - npm install -g react-native-cli
+        - npm install -g detox-cli
+        - yarn
+        - cd e2e
+        - yarn
+        - export PRODUCT_BUNDLE_IDENTIFIER="com.swmansion.rn_gesture_handler_example"
+        - detox build -c ios.sim.release > /dev/null
+      script:
+        - detox test -c ios.sim.release --no-color --headless --cleanup
 
-    script:
-    - detox test -d --no-color --configuration ios.sim.debug --cleanup
+    - language: android
+      dist: trusty
+      sudo: required
+      jdk: oraclejdk8
+      env: NODE_VERSION=10.11.0
+      before_cache:
+        - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+        - rm -rf $HOME/.gradle/caches/*/plugin-resolution/
+      cache:
+        directories:
+          - $HOME/.yarn-cache
+          - $HOME/.gradle/caches/
+          - $HOME/.gradle/wrapper/
+      android:
+        components:
+          - tools
+          - platform-tools
+          - build-tools-28.0.3
+      install:
+        - yes | sdkmanager "platforms;android-28"
+        - nvm install $NODE_VERSION
+        - nvm use $NODE_VERSION
+        - nvm alias default $NODE_VERSION
+        - node --version
+        - npm install -g yarn
+        - npm install -g react-native-cli
+        - npm install -g detox-cli
+        - cd Example
+        - yarn
 
-  - language: android
-    sudo: required
-    jdk: oraclejdk8
-    env: NODE_VERSION=10.11.0
-    before_cache:
-    - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-    - rm -rf $HOME/.gradle/caches/*/plugin-resolution/
-    cache:
-      directories:
-      - $HOME/.yarn-cache
-      - $HOME/.gradle/caches/
-      - $HOME/.gradle/wrapper/
-    android:
-      components:
-      - tools
-      - platform-tools
-      - build-tools-27.0.3
+      script:
+        - cd ./android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug
```

Comment:
@osdnk Why this command has `assembleAndroidTest -DtestBuildType=debug` is it some stuff that was relevant back in the old days when detox worked with android?

Reply:
Because I think we might want to remove it and just test if build is passing for debug app 🤔